src/hooks/index.ts

```diff
@@ -48,4 +49,28 @@ export const useEventResponses = () : EventResponses => {
     }
 
     return { eventResponses, onSignup, onUndoSignup };
+};
+
+type RespondEvents = {
+    onUndoSignup: OnUndoSignup;
+    respondEvents: ZetkinEvent[] | undefined;
+}
+
+export const useRespondEvents = () : RespondEvents => {
+    const respondEventsQuery = useQuery('respondEvents', getRespondEvents());
+    const respondEvents = respondEventsQuery.data;
+
+    const queryClient = useQueryClient();
+
+    const removeFunc = useMutation(deleteEventResponse, {
+        onSettled: () => {
+            queryClient.invalidateQueries('respondEvents');
+        },
+    });
+
+    function onUndoSignup (eventId : number, orgId : number) {
+        removeFunc.mutate({ eventId, orgId });
+    }
+
+    return { onUndoSignup, respondEvents };
```

Comment:
Isn't this just duplicating functionality that is already achieved with `useEventResponses()`?

Reply:
I did this for two reasons:
1. `MyTodoPage` doesn't need the `eventResponsesQuery` 
2. It felt more direct to invalidate `respondEvents` instead of `eventResponses` on `MyTodoPage`.

At some point I guess `respondEvents` and `eventResponses` will be the same, but I didn't have the capacity of figuring that out at this point.